src/db.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+
+class DbConnection {
```

Comment:
You can rename this class to `Database` because when you're importing this in app.js that's the name you've used. Go for consistency always!